dora/integration/fuse/bin/alluxio-fuse

```diff
@@ -329,7 +329,7 @@ unmount_command() {
   readonly force_kill
 
   local fuse_pid
-  fuse_pid=$(ps ax -o pid,args | grep [A]lluxioFuse | grep " ${mount_point} " | awk '{print $1}')
+  fuse_pid=$(ps ax -o pid,args | grep [A]lluxioFuse | grep " ${mount_point}" | awk '{print $1}')
```

Comment:
this may not help distringuish "/path1" from "/path1/path2"

Reply:
actually i know why this isn't working anymore, its bcos of the mnt table mgr change, the start up script transfers from 
`-m <mount_point> -u <ufs_root_path>`
to 
`m <mount_point>`
if mnt table mgr is enabled, hence not working, should find a way to correct this due to mnt table mgr change. 
@jiacheliu3  @dbw9580   FYI

Reply:
using word boundary for this